### ISSUE TITLE
Contact Page QA

### ISSUE THREAD

**Issue by vickibrown** [thinkshout/thinkshout#826]
https://thinkshout.com/contact/

This page needed more work so I made a quick mock-up. Note that the h1 and h2 don't need to change, I just included in them in the mock-up for context.

https://thinkshout.invisionapp.com/share/F3QCZY3ZBDW#/416471725_Contact_Desktop

**Comment by vickibrown** [comment]
@komejo just one thing. Can you make the labels above the fields a little heavier in weight?

**Comment by JulesKhong** [comment]
Vicki's design looks great. Ideally folks tab to the subscribe button before the submit. @vicki would it be possible to move it above the button instead of after it?

@komejo A few things to look out for: programmatically related labels on the fields, and our phone number should probably be a link. I'll hold on any other feedback since there are a lot of design changes.

**Comment by vickibrown** [comment]
@JulesKhong yes to moving the button. For the text link lets just keep it the same gray color as the other text and underline it.

**Comment by komejo** [comment]
Fixed and ready for final QA: https://a11y-2020--thinkshout.netlify.app/contact/

**Comment by JulesKhong** [comment]
- [ ] The orange link text still needs to be updated, and the phone number should be a link https://github.com/thinkshout/thinkshout/issues/826#issuecomment-636247920

**Comment by komejo** [comment]
@JulesKhong I think the link text was cached, but good catch on the phone #. This is what we're expecting, right? 

![Screen Shot 2020-06-01 at 6 57 20 PM](https://user-images.githubusercontent.com/1668324/83471538-c4cbc080-a439-11ea-958b-48612f51e84a.png)

**Comment by vickibrown** [comment]
@komejo looks good to me.

**Comment by JulesKhong** [comment]
Looks great, thanks @komejo!
